docs/modules/datacleaning.md

```diff
@@ -3,42 +3,44 @@ title: Data Cleaning
 ---
 # Data Cleaning
 
-Data Cleaning An Omeka S module for low-level auditing and cleaning of resource metadata. It is designed to be used to prepare resource metadata for use in visualizations.
+The [Data Cleaning module](https://omeka.org/s/modules/DataCleaning){target=_blank} is for low-level auditing and cleaning of resource metadata. It is designed to prepare resource metadata for use in visualizations.
 
-Changes made in Data Cleaning cannot be easily undone, and can be destructive. Due to the powerful nature of the module, it can only be used by Global Administrator users. 
+Changes made by Data Cleaning cannot be easily undone, and can be destructive. Due to the powerful nature of the module, it can only be used by Global Administrators. 
 
-**Before running any audit, make sure you have a recent backup of your data.** 
+!!! note
+	Before running any audit, make sure you have a recent backup of your data.
```

Comment:
I'm going to leave this for now - we won't put it in the module page but I'll make a note for myself to add database-backup instructions to some of the more basic pages.

Reply:
A "pithy" command for doing the backup is a little tricky just because the specifics of peoples' databases differ.

Generally I'd say if you're talking command line that

    mysqldump -u *your db user* -p --single-transaction --default-character-set=utf8mb4 *your omeka dbname* > *filename of backup.sql*

would be the way to go. Many people are going to have a cPanel button they can just push, or could do it in phpMyAdmin, or something along those lines, which they're probably much more likely to actually do.